common/unsafe/src/main/java/org/apache/spark/unsafe/types/CalendarInterval.java

```diff
@@ -82,40 +70,6 @@ public int hashCode() {
     return Objects.hash(months, days, microseconds);
   }
 
-  @Override
-  public String toString() {
```

Comment:
Let's not move this. `toString` is a very basic functionality and we need it even after we expose `CalendarInterval`

Reply:
revert done，thanks for helping me.